### ISSUE TITLE
Static zimit2 archives do not display properly in Restricted mode

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1476]
In the PWA we do some processing of static zimit2 archives so that they show at least non-dynamic elements in Restricted mode, and this works well enough for previewing such archives in a sandboxed environment.

In this app, we do that processing for zimit classic archives (in Restricted mode), but ironically not for zimit2 where many of the links are in fact pre-processed, so the transformations should be easier.

The code is available, so it is worthwhile backporting here.
